packages/libp2p/src/connection-manager/index.ts

```diff
@@ -237,7 +239,7 @@ export class DefaultConnectionManager implements ConnectionManager, Startable {
       logger: components.logger
     }, {
       maxConnections: this.maxConnections,
-      allow: this.allow
+      allow: init.allow != null ? init.allow.map(a => multiaddr(a)) : undefined
```

Comment:
Slightly more concise:

```suggestion
      allow: init.allow?.map(a => multiaddr(a))
```